web/src/components/molecules/Asset/Asset/AssetBody/Asset.tsx

```
@@ -66,67 +66,72 @@ const AssetMolecule: React.FC<Props> = ({
         );
       case "csv":
         return (
           <CsvViewer
+            isAssetPublic={asset.public}
             url={assetUrl}
-            onGetViewer={onGetViewer}
             workspaceSettings={workspaceSettings}
+            viewerRef={viewerRef}
           />
         );
       case "unknown":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)